array-manipulation-3/src/unique.js

```diff
@@ -0,0 +1,35 @@
+/* exported unique */
+// create a function to return a new array with no repeating values
+// create a new empty array
+// create a variable to hold a boolean
+// loop through the array
+// push the first value without checking the secon array
+// loop through the second array
+// check if an item in the first array is equal to any item in the second
+// if it is, change the boolean to false
+// else at the end of the loop push the value to the new array
+// change the boolean back to true
+// return the new array from the function containing no repetitive  values.
+
+function unique(array) {
+  var newArr = [];
+  var found = true;
+  for (var i = 0; i < array.length; i++) {
+    found = true;
+    if (i === 0) {
+      newArr.push(array[i]);
```

Comment:
Rather than doing this, you can just start with the value at the `0` index in the array, then begin the loop at `1` 🤓